Dockerfile

```diff
@@ -43,6 +44,10 @@ RUN --mount=type=cache,target=/go/pkg/mod,uid=10001,gid=10001 \
     --mount=type=cache,target=/home/dev-user/.cache/go-build,uid=10001,gid=10001 \
     make $(echo ${TARGETPLATFORM} | tr '/' '_')
 
+# Download latest versions of Leaflet, htmx, Alpine.js and Tailwind CSS
+RUN make download-assets
```

Comment:
How come you are adding these after building the birdnet-go binary? This means that these are redownloaded for no reason every time the image is rebuilt.

To me, the only use-case for these two lines appears to be inside the devcontainer itself. However, the devcontainer does not even use this stage. Since it uses the buildenv stage which is the one above this stage.

I do think getting the bundled packages such as tailwind and alpinejs etc out of the source code and into some sort of package manager does make sense. But currently, it seems to only be used inside the devcontainer, since the actual binary will be built with whatever was in the assets folder once the repo was cloned.